plugins/woocommerce/tests/e2e-pw/utils/editor.js

```diff
@@ -10,8 +10,8 @@ const closeWelcomeModal = async ( { page } ) => {
 };
 
 const disableWelcomeModal = async ( { page } ) => {
-	await page.waitForLoadState();
-	await page.waitForFunction( () => window?.wp?.data );
+	await page.waitForLoadState( 'domcontentloaded' );
+	// await page.waitForFunction( () => window?.wp?.data );
```

Comment:
I'm not sure how this change helps with flakiness.

Reply:
The test was failing consistently on this line -- I think that sometimes this function was failing/failing to load/I don't know. I tried a few different wait scenarios and waiting for the DOM seemed to work best.

Reply:
If `window?.wp?.data` is not available the next code block will fail with a less friendlier error - the reason I've added this particular check. `window?.wp?.data` not being available most probably means the page did not load in the expected time. 
I guess waiting for `domcontentloaded` does the same thing, `domcontentloaded` probably implies that `window?.wp?.data` also loaded, as opposed to the default `load`. 
I see an advantage in having a single line instead of two, but still don't see why it helps with flakiness. 🤔 
If you decide to leave it as is, can you please remove the commented out code?

Reply:
I've seen a number of failures happen on this line.  

I'm not sure that this is the right fix, but let's give it a try for now.  I'll remove the commented line, but put in a comment so that if we continue to see errors, the next person investigating will know what went down.